src/SpaceKitProvider/index.tsx

```diff
@@ -9,10 +9,13 @@ interface State {
    * @default false
    */
   disableAnimations: boolean;
+
+  theme: "light" | "midnight";
```

Comment:
Can we use `dark` instead of `midnight` to keep it consistent with our theme names everywhere else?

Reply:
Thanks for starting this conversation. After talking with @jglovier we should rename this to `dark` because using midnight is an implementation detail 👍

Reply:
Thanks for starting this conversation @Jephuff ! After talking with @jglovier we should rename this to `dark` because using midnight is an implementation detail 👍

Reply:
Done.